### PR TITLE
Fixes restore experiment error [fixes SCI-464]

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-464

### CODE REVIEW

File: app/controllers/experiments_controller.rb [hunk 1]
```diff
@@ -76,16 +76,24 @@ def update
...
       respond_to do |format|
         format.json do
           render json: {}, status: :ok
         end
+        format.html do
+          redirect_to project_path(@experiment.project)
+        end
       end
     else
+      flash[:alert] = t('experiments.archive.restore_flash')
```
Comment (mz3944) [inline app/controllers/experiments_controller.rb]: Here also other update experiment errors can occur, not just when restoring archived experiment. You can make a check if restoration is in progress, and based on that show flash error.

Comment (mz3944) [inline app/controllers/experiments_controller.rb]: Just use 'experiments.update.error_flash' instead.

Comment (mz3944) [inline app/controllers/experiments_controller.rb]: Just use 'experiments.update.error_flash' instead.

**Comment by ZmagoD** [comment]
is ok now

**Comment by mz3944** [comment]
http://www.relatably.com/m/img/elegant-memes/cmtc36.jpg

:branch => :merge

**Pull request merged**